test/unit/helpers/pages_helper_test.rb

```diff
@@ -10,7 +10,7 @@ class PagesHelperTest < ActionView::TestCase
     setup do
       @rubygem = create(:rubygem, name: "rubygems-update")
       @version_first = create(:version, number: "1.4.8", rubygem: @rubygem)
-      @version_last = create(:version, number: "2.4.8", built_at: DateTime.new(2015,1,1), rubygem: @rubygem)
+      @version_last = create(:version, number: "2.4.8", built_at: DateTime.new(2015, 1, 1), rubygem: @rubygem)
```

Comment:
I just fixed this in `master` too. You should be able to rebase to pick up that change.